docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -158,6 +166,20 @@ just pass -Dquarkus.package.type=native as a property on the command line, howev
 this allows native image tests to also be run.
 
 Create a native executable using: `./mvnw package -Pnative`.
+[[graal-and-windows]]
+[NOTE]
+.Issues with packaging on Windows
+====
+The Microsoft Native Tools for Visual Studio must first be initialized before packaging. You can do this by starting
+the `x64 Native Tools Command Prompt` that was installed with the Visual Studio Build Tools.
```

Comment:
@gsmet Wording here will be a little odd as well with the Windows 7 SDK vs Visual Studio 2017 Build Tools.